app/test_main.py

```diff
@@ -1,3 +1,35 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (10, 10, [0, 0]),
+        (15, 15, [1, 1]),
+        (20, 20, [1, 1]),
```

Comment:
It seems that the test cases for cat and dog ages 15 and 20 both expect the same output [1, 1]. According to the checklist, make sure that your tests don't test the same case multiple times. Consider removing one of these test cases or adjusting the expected output if they are meant to test different scenarios.